cats-effect-3/src/main/scala/parallelfor/interop/cats/package.scala

```diff
@@ -0,0 +1,22 @@
+package parallelfor.interop
+
+import _root_.cats.effect.IO
+import parallelfor.{Parallelizable, Parallelizable1}
+
+package object cats {
+
+  implicit val ioPar1: Parallelizable1[IO] = new Parallelizable1[IO] {
```

Comment:
in theory, we should be able to define this for any `F[_]: Parallel`, then we don't even need a dependency on CE (this is a Cats typeclass). I wasn't able to make it work that way though, with the macros, variance and all :/

Reply:
Aw! Well thanks for giving it a whirl! Macros, HKTs, and Variance is recipe for weird compiler behavior.